test/test_visualization.jl

```diff
@@ -32,7 +32,8 @@ test_examples_2d = Dict("TreeMesh" => ("tree_2d_dgsem",
                                         "elixir_euler_source_terms_nonconforming_unstructured_flag.jl"),
                         "DGMulti" => ("dgmulti_2d", "elixir_euler_weakform.jl"))
 
-@testset "PlotData2D, PlotDataSeries, PlotMesh with $mesh" for mesh in keys(test_examples_2d)
+@testset "PlotData2D, PlotDataSeries, PlotMesh with $mesh" for mesh in
+                                                               keys(test_examples_2d)
```

Comment:
Is this done by the formatter?

Reply:
@midsorbet See https://trixi-framework.github.io/TrixiDocumentation/stable/styleguide/#automated-source-code-formatting for a description of the automatic source code formatter we use here